plugin/kubernetes/README.md

```diff
@@ -101,7 +102,9 @@ kubernetes [ZONES...] {
 * `ignore empty_service` returns NXDOMAIN for services without any ready endpoint addresses (e.g., ready pods).
   This allows the querying pod to continue searching for the service in the search path.
   The search path could, for example, include another Kubernetes cluster.
-
+* `wildcards enabled|disabled` enables or disables the wildcard query feature (see Wildcards section below).
+  Default is enabled.
```

Comment:
Hmm, another option that requires deprecating. I wonder can't we just rip the wildcard stuff out at once and be done with it?